### PR TITLE
add .gitattributes for LF/CRLF problems, Material Icon/Script, reset CSS

### PR DESCRIPTION
**Pull request by trungh13**
1. add .gitattributes for LF/CRLF problems( between Mac / Windows user)  following https://eslint.org/docs/rules/linebreak-style
2. Material Icon/Script for Materialize-css
3. Reset CSS ( Margin 0, Padding 0, Box-sizing)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+::root{
+  --color-1:#FF8000;
+  --color-2:#FF9400;
```
[the remark this answers is not in the document]
Comment (Nizar-Rahme) [reply src/App.css]: @christianhamamdjian can you suggest names for the variables? and add shades like: `--primary-dark`, `--primary-light`

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+::root{
```
Comment (Nizar-Rahme) [inline src/App.css]: root has only one colon `:root`

File: src/App.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ditiite) [reply src/App.css]: **--primary** -> A primary color is the color displayed most frequently across your app’s screens and components.

**--primary-dark** 
**--primary-light**  -> To create a contrast between UI elements, such as distinguishing a top app bar from a system bar, use light or dark variants of primary color on each element. Can also use variants to distinguish elements within a component, such different variants used on a floating action button container, and the icon within it

**--secondary**
**--secondary-dark**
**--secondary-light**  -> Secondary colors are best for:
-Floating action buttons
-Selection controls, like sliders and switches
-Highlighting selected text
-Progress bars
-Links and headlines

*_Surface, background, and error colors typically don’t represent brand:_*
**--surface** -> affect surfaces of components, such as cards, sheets, and menus.
**--background** -> appears behind scrollable content. The baseline background and surface color is #FFFFFF.
**--error** -> indicates errors components, such as text fields. The baseline error color is #B00020.


*_“On” colors are primarily applied to text, iconography, and strokes. Sometimes, they are also applied to surfaces. Defaults for these colors are #FFFFFF and #000000_*
**--on-dark** -> color for text, icon, strokes used on dark background.
**--on-light** -> color for text, icon, strokes used on light background.

**Comment by trungh13** [comment]
Dita fixed the colors, and also  colons typos

**Pull request merged**